src/main/res/values/strings.xml

```diff
@@ -32,18 +32,18 @@
   <string name="link_auth_sub_title">Do you want to approve?</string>
   <string name="link_auth_message">%1$s</string>
   <string name="establishing">Establishing secure connection</string>
-  <string name="cancel_sync">Cancel Sync</string>
+  <string name="cancel_sync"><u>Cancel Import…</u></string>
```

Comment:
did you know that some devices don't render the html tags? they render the tag as text